docs/querying/datasource.md

```diff
@@ -211,6 +219,50 @@ Inline datasources are not available in Druid SQL.
 Refer to the [Query execution](query-execution.md#inline) page for more details on how queries are executed when you
 use inline datasources.
 
+### Dynamic table append `TABLE(APPEND(...))`
+
+This is essentially an SQL level syntactic sugar which matches columns by name from multiple tables.
+Suppose you have 3 tables:
+* `table1` has `column1`
+* `table2` has `column2`
+* `table3` has `column1`, `column2`, `column3`
+
+It is possible to create an union view of the above tables is possible by using UNION ALL:
+```sql
+SELECT column1,NULL AS column2,NULL AS column3 FROM table1
+UNION ALL
+SELECT NULL AS column1,column2,NULL AS column3 FROM table2
+UNION ALL
+SELECT column1,column2,column3 FROM table3
+```
+
+However depending on the size of the table's schema it might be quite complicated to do that; `TABLE(APPEND('table1','table2','table3'))` represents the same in a more compact form.
+
+<Tabs>
```

Comment:
All the sections that have these tabs do so at the beginning of the section, so consider moving this to the top for consistency